snooty.toml

```diff
@@ -22,7 +22,7 @@ driver-long = "MongoDB Kotlin Driver"
 version = "4.11"
 full-version = "{+version+}.0"
 mdb-server = "MongoDB server"
-kotlin-docs = "https://kotlinlang.org/docs"
+kotlin-docs = "https://kotlinlang.org"
```

Comment:
just curious about the reason for this change

Reply:
So that it is usable for links that are at `https://kotlinlang.org/docs` and `https://kotlinlang.org/api/`, seemed silly to use the source constant for some links at the same site but not others